### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by Olenka-Hryk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,54 @@ dist/
 nbdist/
```
Comment (af150416) [inline .gitignore:27]: You need to remove all unnecessary files from git !!!
It is very hard to review the code when so much rubbish in repository

Comment (af150416) [reply .gitignore:27]: and do the pulls much more often

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/components/entity/Cheque.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
+package com.store.components.entity;
+
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/components/entity/Cheque.java:2]: docs and more docs, everywhere (remember, that  only you know, what this code for)

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	 * variable to work with the base
+	 */
+	@Autowired
+	private JdbcTemplate jdbc;
+
+	public static class workingWithRowMap implements RowMapper<Cheque> {
+		/**
+		 * copy data from DB to cheque
+		 */
+		public Cheque mapRow(ResultSet rs, int rowNum) throws SQLException {
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java:28]: may be using BeanPropertyRowMapper will be more laconic

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+			Cheque cheque = new Cheque();
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java:29]: anywhere, if you have exact constructor, why do you ignore it The same for other entities creation.

File: GroceryStoreRESTfullCRUD/build.gradle [hunk 5]
```diff
@@ -32,6 +34,10 @@ dependencies {
 	compile('org.springframework.boot:spring-boot-starter-jdbc')
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/build.gradle:34]: use hard versions always and everywhere,only imagine what will happen when somebody use your code with unsupported libraries

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+	public void addCheque(Cheque cheque) {
+		final String SQL = "INSERT INTO Cheque (idReserve, idOrder, idCustomer, idSale, Amount, OrderDate) values (?, ?, ?, ?, ?, ?) ";
+		final int idReserve = cheque.getIdReserve();
+		final int idOrder = cheque.getIdOrder();	
+		final int idCustomer = cheque.getIdCustomer();
+		final int idSale = cheque.getIdSale();
+		final float amount = cheque.getAmount();
+		final String orderDate = cheque.getOrderDate();
+
+		jdbc.update(SQL, new Object[] { idReserve, idOrder, idCustomer, idSale, amount, orderDate });
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java:70]: do you really need this new Object[]{}? The same for other queries.

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+	 */
+	public Cheque infoAboutAmountOfOrderById(int id) {
+		final String SQL = "SELECT * from Cheque WHERE idOrder=?";
+		Cheque cheque = jdbc.queryForObject(SQL, new workingWithRowMap(), id);
+		return cheque;
+	}
+	
+	public void addCheque(Cheque cheque) {
+		final String SQL = "INSERT INTO Cheque (idReserve, idOrder, idCustomer, idSale, Amount, OrderDate) values (?, ?, ?, ?, ?, ?) ";
+		final int idReserve = cheque.getIdReserve();
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java:63]: why we need to create all this variables?

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/components/entity/Cheque.java [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+	private int idOrder;
+	private int idCustomer;
+	private int idSale;
+	private float amount;
+	private String orderDate;
+
+	public Cheque() {
+	}
+
+	public Cheque(int id, int idReserve, int idOrder, int idCustomer, int idSale, float amount, String orderDate) {
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/components/entity/Cheque.java:15]: is this constructor necessary? Do you somewhere use it? Same for other entities.

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+			cheque.setAmount(rs.getInt(6));
+			cheque.setOrderDate(rs.getString(7));
+
+			return cheque;
+		}
+	}
+	
+	public Collection<Cheque> infoAboutAllCheque(int page) {
+		page *= 3;
+		final String SQL = "SELECT * FROM Cheque LIMIT 3 offset ?";
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/components/service/ChequeDao.java:44]: Remove this final variable from the method The same for other final variables.

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/controller/ChequeController.java [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.store.components.entity.Cheque;
+import com.store.service.ChequeService;
+
+@RestController
+@RequestMapping("/store/cheque")
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/controller/ChequeController.java:16]: mapping must to fit the standard pattern for rest services version one "/rest/project/v1/constructor/method"

File: GroceryStoreRESTfullCRUD/src/main/java/com/store/controller/ChequeController.java [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+	 * Get amount cheque from DB
+	 * 
+	 * @return
+	 */
+	@RequestMapping(value = "/view_amount/{id}", method = RequestMethod.GET)
+	public Cheque infoAboutAmountOfOrderById(@PathVariable("id") int id) {
+		return chequeService.infoAboutAmountOfOrderById(id);
+	}
+
+	@RequestMapping(value = "/create", method = RequestMethod.POST)
```
Comment (af150416) [inline GroceryStoreRESTfullCRUD/src/main/java/com/store/controller/ChequeController.java:41]: "/create" is unnecessary, mapping already unique by request method

**Review by koropatva — COMMENTED** [review]
pls clean project from all unnecessary files. Why we need all those `/bin` and other folders?

**Pull request closed without merging**